README.md

```diff
@@ -113,244 +112,128 @@ bash test/test_cleanup.sh
 Now that your virtual environment is set up and the workflow is deployed and
 tested, you can go ahead and run the workflow on your samples.
 
-But first, here is a brief description of what the workflow does:
-
-### Workflow description
-
-The workflow consist on one main `Snakefile` 
-gathering three different subworkflows,
-**_PREPARE_**, **_MAP_** and **_QUANTIFY_**. 
-Each one of them is implemented in an individual Snakemake file
-and fully containerized.    
-
-
-#### _PREPARE_
-
-The first subworkflow, **_PREPARE_**, downloads and processes "genome resources"
-from the publicly available repositories [Ensembl][ensembl] and
-[miRBase][mirbase] according to your instructions. Resources are then processed
-to prepare indexes and other contingent resources that will be used in later
-steps.
-
-#### _MAP_
-
-The second subworkflow, **_MAP_**, aligns the user-provided short read smallRNA-seq
-libraries against the references generated with the **_PREPARE_** subworkflow. For
-increased fidelity it uses two separate aligning tools, [Segemehl][segemehl]
-and our in-house tool [Oligomap][oligomap]. In both cases, reads are aligned
-separately to the genome and the transcriptome. Afterwards, alignments are
-merged in a way that only the best alignment (or alignments) of each read are
-kept.
-
-#### _QUANTIFY_
-
-The third and final subworkflow, **_QUANTIFY_**, quantifies miRNA expression by
-intersecting the alignments from the **_MAP_** subworkflow with the annotations
-generated in the **_PREPARE_** subworkflow. Intersections are computed with
-[`bedtools`][bedtools] for one or multiple of mature, primary transcripts and
-isomiRs. Reads consistent with each miRNA are counted and tabulated.
-
-
-The schema below is a visual representation of
-an example run of the whole workflow:
-
-> ![rule-graph][rule-graph]
-
-
-### Running the workflow
-
 Assuming that you are currently inside the repository's root directory, 
-create a directory from which you will run your workflow and
-name it whatever you want e.g., `MY_ANALYSIS` and 
-head to it.
+create a directory from which you will run your workflow and name it whatever 
+you want e.g., `MY_ANALYSIS` and head to it.
 
 ```bash
 mkdir MY_ANALYSIS
 cd MY_ANALYSIS
 ```
-Now within the `MY_ANALYSIS/` directory, 
-create a new directory to store you library; 
-name it as you like, e.g. `my_lib` and move there.    
+Place your library file(s) here. In addition, create the sample table 
+`sample_table.csv` and fill it up. The `sample_table.csv` is a tab-separated  
```

Comment:
There's no reason to dictate a name for the sample table. They can call it `i_name_my_files_what_i_want.i_dont_care_about_file_extensions_either` and that's totally fine.